Snail/Observable.swift

```diff
@@ -6,7 +6,7 @@ import Dispatch
 public class Observable<T>: ObservableType {
     private var isStopped: Int32 = 0
     private var stoppedEvent: Event<T>?
-    private var subscribers: [Subscriber<T>] = []
+    private(set) var subscribers: [Subscriber<T>] = []
```

Comment:
Is this for testing purposes?

Reply:
yeah I needed to change the getter to internal to run your unit test.